qutebrowser/mainwindow/tabwidget.py

```diff
@@ -705,7 +705,19 @@ def wheelEvent(self, e):
             e: The QWheelEvent
         """
         if config.val.tabs.mousewheel_switching:
-            super().wheelEvent(e)
+            if utils.is_mac:
+                # WORKAROUND for this not being customizable until Qt 6:
+                # https://codereview.qt-project.org/c/qt/qtbase/+/327746
+                index = self.currentIndex()
+                if index == -1:
+                    return
+                delta = e.angleDelta().x() if abs(e.angleDelta().x()) > abs(e.angleDelta().y()) else e.angleDelta().y()
```

Comment:
This line is over the line length limit now - I'd probably do something like:

```suggestion
                dx = e.angleDelta().x()
                dy = e.angleDelta().y()
                delta = dx if abs(dx) > abs(dy) else dy
```